api/server.go

```diff
@@ -870,6 +884,8 @@ func NewServer(e Explorer, cm ChainManager, s Syncer, ex exchangerates.Source, a
 
 		"POST   /hosts": srv.hostsHandler,
 
+		"GET    /proof/:index": srv.proofIndexHandler,
```

Comment:
This route is not terribly descriptive. It definitely short and sweet but it's not very user friendly. Should we consider `leaves/:index/proof` or `/state/tree/proofs/:index`? Feel free to resolve though if you disagree.